src/browser/data/technologies.json

```diff
@@ -1,105 +1,150 @@
 {
   "0":[
     {
-      "id" : 8,
-      "name" : "scala",
+      "id" : 1,
+      "name" : "Scala",
       "src" : "assets/img/backend/scala.png"
     },
     {
-      "id" : 9,
+      "id" : 2,
       "name" : "akka",
       "src" : "assets/img/backend/akka.png"
     },
     {
-      "id" : 10,
+      "id" : 3,
       "name" : "play",
       "src" : "assets/img/backend/play.png"
     },
     {
-      "id" : 11,
+      "id" : 4,
       "name" : "slick",
       "src" : "assets/img/backend/slick.png"
     },
     {
-      "id" : 12,
+      "id" : 5,
       "name" : "spray",
       "src" : "assets/img/backend/spray.png"
+    },
+    {
+      "id" : 6,
```

Comment:
In fact, I'm not sure, it looks like it's used in render so probably it's not necessary. Firstly I was thinking that it's used on backend to differentiate so I kept it but now I know there is going only skill name.